andela/taps-client
# #160928262 Test files in Home module

#### What does this PR do?
-  adds tests for `CardItem` component
-  adds tests for `Cards` component
-  adds tests for `Home` container

#### Description of Task to be completed?
- Add tests for the files in modules/Home folder

#### How should this be manually tested?
- run `git pull` command
- run `npm test`
- the coverage report should show that `modules/Home` has 100% coverage

#### Any background context you want to provide?
- the files in the `modules/Home` folder have not yet been tested. This `PR` adds tests for them


#### What are the relevant pivotal tracker stories?
#160928262 

#### Screenshots or gifs (if appropriate)
<img width="1440" alt="screen shot 2018-10-11 at 9 37 05 am" src="https://user-images.githubusercontent.com/38565349/46791527-43da2580-cd39-11e8-9583-6e41211f9198.png">
<img width="1436" alt="screen shot 2018-10-11 at 9 36 50 am" src="https://user-images.githubusercontent.com/38565349/46791529-43da2580-cd39-11e8-998b-ad6ed618e61d.png">

#### Questions:
- none
